### DISCUSSION TITLE
Incoming Message Logging

### DISCUSSION THREAD

**Discussion by iammattmartin** [postalserver/postal#1554] [Help with using Postal]
Since upgrading to v2 (new server, database move), we are not seeing any incoming messages logged.

They are being processed and forwarded to the correct SMTP route.

Outgoing messages are still being logged.

Is there anywhere obvious to check? postal logs seems to filter through days of logs with no -f / -t command to only see the latest as far as I can spot.

**Comment by willpower232** [comment]
Do you mean that the messages are being processed by Postal but don't appear in the UI?

**Reply by iammattmartin** [reply]
Yes, everything else is there (i.e outgoing) but no incoming messages are logged.

**Reply by willpower232** [reply]
Can you find them in the database?
